.vscode/launch.json

```diff
@@ -0,0 +1,60 @@
+{
+    "configurations": [
+        {
+            "name": "Debug cloe",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${workspaceFolder}/build/RelWithDebInfo/bin/cloe-engine",
```

Comment:
`RelWithDebInfo` is a variable we can't control.

Reply:
I would use ``CLOE_ENGINE`` env variable to get the path to the used cloe-engine executable